src/modules/Elsa.Hosting.Management/Options/HeartbeatSettings.cs

```diff
@@ -0,0 +1,7 @@
+namespace Elsa.Hosting.Management.Options;
+
+public class HeartbeatSettings
```

Comment:
Although I prefer the use of the "Settings" suffix, everywhere else we use "Options" as the suffix - a choice that I regret in hindsight. On the one hand I would like to keep using "Settings" as introduced here, but on the other hand, it might be better to remain consistent and use "Options", and as a separate refactoring task, rename everything from ...Options to ...Settings. What do you think?

Reply:
I used settings since I saw another file use it as well. Since the vast majority is named Options let's keep it like that for now.
Although I do like the suffix settings a bit better I don't think it warrants a rename for all options. Especially not in a PR like this.